tasks/skylab/Metrics.wdl

```diff
@@ -1,15 +1,16 @@
 version 1.0
 
-task CellSortBam {
+task CalculateCellMetrics {
   input {
     File bam_input
+    File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
-    Int machine_mem_mb = 100000
-    Int cpu = 2
-    Int disk = ceil(size(bam_input, "Gi") * 8)
-    Int preemptible = 3
+    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v1.0.0"
+    Int machine_mem_mb = 8000
+    Int cpu = 8
+    Int disk = ceil(size(bam_input, "Gi") * 4)
+    Int preemptible = 0
```

Comment:
Why 0 preemptible tries?

Reply:
Still curious if this can be at least 1

Reply:
Changed it to 3. It was set to 0 for testing.